run.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+if [ ! -d "opsfiles" ]; then
+    git clone --depth=1  https://github.com/phanbook/opsfiles.git
+fi
+
+chmod 777 -R public/
+chmod 777 -R content/cache/
+chmod 777 -R content/logs
+
+
+# Now use docker setup for default
+if ! which docker > /dev/null; then
```

Comment:
I propose to use something like
```sh
DOCKER_BIN=`command -v docker 2>/dev/null || true`

if [ x"$DOCKER_BIN" = x ]; then
    // ...
fi
```
 
See for more:
* Why `command -v` instead of `which` https://stackoverflow.com/a/677212/1661465
* Why `x` used here https://stackoverflow.com/a/13864829/1661465